src/js/controllers/chatbot-widget.js

```diff
@@ -115,6 +121,9 @@
                     };
                     this.addToChat(errorMessage);
                 };
+
+                // request opening message from chatbot
+                vm.sendMessage(" ", true);
```

Comment:
Is het de bedoeling dat dit altijd getriggerd wordt? Of moeten we dit misschien als boolean in de config definiëren?

Reply:
Het is idd de bedoeling dat het altijd gebeurt. De chatbot preview in de chatbot authoring omgeving zal dit ook altijd doen. Als het welkomstbericht leeg is dan komt er een lege reply en gaat het niet getoond worden. (Voordeel van deze aanpak is dat je het welkomstbericht kan aan- en afzetten zonder codewijziging door dat veld in te vullen of leeg te maken in de chatbotdefinitie.)

Reply:
Ok, dus dit mag dan ook nog in de NGx chatbot versie gestoken worden, vermoed ik?

Reply:
Inderdaad, die zou ook moeten het welkomstbericht ophalen.